### PR TITLE
Add the section on configuring the plugin, fixed minor markdown errors

### LINKED ISSUE #56: Example of configuring postcss-preset-env with postcss.config.js
The README.md has an exhaustive documentation of configuring postcss-preset-env for popular build tools, but it does not mention that it's possible to use `postcss.config.js`

Maybe it has to be addressed in https://github.com/postcss/postcss-loader instead, please feel free to close this if you don't find int relevant.

**UPD:**

The reasoning for this is that the examples use the notation with the function `postcssPresetEnv(/* options */)`, while it's possible to pass plugins in a bunch of different ways.

### PR DESCRIPTION
**Pull request by halfzebra**
Closes #56

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -328,9 +328,40 @@ postcssPresetEnv({
...
 
+### Configuring
+
+Pass the configuration to the plugin in any way supported by [postcss-load-config](https://github.com/michael-ciniawsky/postcss-load-config).
+
+#### postcss.config.js
+
+```js
+module.exports = {
+  indent: 'postcss',
```
Comment (pascalduez) [inline README.md]: Not needed by default, only in some Webpack setups and such.

File: README.md [hunk 2]
```diff
@@ -328,9 +328,40 @@ postcssPresetEnv({
...
+};
+```
+
+#### package.json
+
+```json
+{
+  "postcss": {
+    "postcss-preset-env": {
+      "": 3,
```
Comment (jonathantneal) [inline README.md]: I think this should be `"stage"` instead of `""`.

**Comment by halfzebra** [comment]
Thanks for the review!

I have addressed the comments, please let me know if anything else needs attention.

**Comment by jonathantneal** [comment]
@halfzebra, I think this looks great. @pascalduez, if you approve, I’ll merge.

**Comment by pascalduez** [comment]
@jonathantneal Sure! thanks.

**Comment by jonathantneal** [comment]
Thanks, @halfzebra !

**Pull request merged**